openhtf/core/test_record.py

```diff
@@ -69,12 +69,22 @@ def add_outcome_details(self, code, description=''):
     self.outcome_details.append(OutcomeDetails(code, description))
 
 
+# PhaseResult enumerations are converted to these outcomes by the PhaseState.
+PhaseOutcome = Enum(  # pylint: disable=invalid-name
+    'PhaseOutcome', [
+        'PASS',  # CONTINUE with allowed measurement outcomes.
+        'FAIL',  # CONTINUE with failed measurements.
+        'SKIP',  # SKIP.
+        'ERROR',  # Any terminal result.
+    ])
+
+
 class PhaseRecord(  # pylint: disable=no-init
     mutablerecords.Record(
         'PhaseRecord', ['descriptor_id', 'name', 'codeinfo'],
         {'measurements': None,
          'start_time_millis': int, 'end_time_millis': None,
-         'attachments': dict, 'result': None})):
+         'attachments': dict, 'result': None, 'outcome': None})):
```

Comment:
what's the distinction between 'result' and 'outcome' here?  i think you meant to rename result to outcome?  if not, a comment would be good to differentiate them.